packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -291,15 +323,28 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                                     </Popover>
                                 </div>
                             )}
-                            <div onClick={onClickEdit}>
-                                <Popover
-                                    alignment='bottom'
-                                    className='p2p-my-ads__table-popovers__edit'
-                                    message={localize('Edit')}
-                                >
-                                    <Icon icon='IcEdit' size={16} />
-                                </Popover>
-                            </div>
+                            {!!is_advert_active && (
+                                <div onClick={onClickEdit}>
+                                    <Popover
+                                        alignment='bottom'
+                                        className='p2p-my-ads__table-popovers__edit'
+                                        message={localize('Edit')}
+                                    >
+                                        <Icon icon='IcEdit' size={16} />
+                                    </Popover>
+                                </div>
+                            )}
+                            {!is_advert_active ? (
+                                <div onClick={onClickEdit}>
+                                    <Popover
+                                        alignment='bottom'
+                                        className='p2p-my-ads__table-popovers__edit'
+                                        message={localize('Edit')}
+                                    >
+                                        <Icon icon='IcEdit' size={16} />
+                                    </Popover>
+                                </div>
+                            ) : null}
```

Comment:
Same mysterious case as the one above. Aren't these the same element? So regardless of the conditions one will show

Reply:
Oops.. Will refactor the code